src/main/java/labs/introtoprogramming/lab5/object/Parser/PolygonObject.java

```diff
@@ -0,0 +1,53 @@
+package labs.introtoprogramming.lab5.object.Parser;
+
+import labs.introtoprogramming.lab5.geometry.Vector3;
+
+import java.util.ArrayList;
+
+public class PolygonObject {
```

Comment:
It is better to rename it to Polygon and PolygonObject will extends SceneObject and store all polygongs. Also it is better to replace polygon in object with triangle.